mtags/src/main/scala-2/scala/meta/internal/pc/completions/InterpolatorCompletions.scala

```diff
@@ -127,15 +126,12 @@ trait InterpolatorCompletions { this: MetalsGlobal =>
       out.toString
     }
 
-    val filter: String =
-      text.substring(lit.pos.start, pos.point - interpolator.name.length)
-
     override def contribute: List[Member] = {
       metalsScopeMembers(pos).collect {
         case s: ScopeMember
             if CompletionFuzzy.matches(interpolator.name, s.sym.name) =>
           val edit = new l.TextEdit(nameRange, newText(s.sym))
-          val filterText = filter + s.sym.name.decoded
+          val filterText = s.sym.getterName.decoded
```

Comment:
`s.sym.name.decoded` will not be enough ?

Reply:
From what I see this should be enough or is there a particular reason you used `getterName` ?

Reply:
I've just copied its usage from `newText` methods:
https://github.com/scalameta/metals/blob/99367756fe15268a192cd2e194f27e9e2f643fef/mtags/src/main/scala-2/scala/meta/internal/pc/completions/InterpolatorCompletions.scala#L110-L113

I'm don't know if `getterName` usage is required here, but at least it should use `dropLocal` since `s.sym.name.decoded` may contain space at the end.